### PR TITLE
adds news and microforms ILL link

### PR DESCRIPTION
**Pull request by banukutlu**
Issue #644

### CODE REVIEW

File: spec/features/availability_spec.rb [hunk 1]
```diff
@@ -329,5 +329,9 @@
         expect(page).not_to have_link 'Copy unavailable, request via Interlibrary Loan'
       end
     end
+
+    describe 'News & Microforms Scan Request' do
+      it 'display the correct ILL url'
```
Comment (cdmo) [inline spec/features/availability_spec.rb:334]: Why a pending spec here again? Because you plan to write the sinatra fake?

Comment (banukutlu) [reply spec/features/availability_spec.rb:334]: yes, I didn't want to add new fixtures to our sample extract just for these cases. I would need to ask for a new extract for our sample with catkeys for these cases, and then they might change even 🤔

Comment (cdmo) [reply spec/features/availability_spec.rb:334]: ok, marking this as part of #631

File: app/javascript/availability/index.js [hunk 2]
```diff
@@ -406,7 +406,7 @@ const availability = {
 
     generateCallNumber(holding) {
         // Do not display call number for on loan items
-        return (holding.locationID === 'ILLEND') ? "" : holding.callNumber;
+        return (holding.locationID === "ILLEND") ? "" : holding.callNumber;
```
Comment (cdmo) [inline app/javascript/availability/index.js:409]: was this a lint thing? single versus double quotes?

Comment (banukutlu) [reply app/javascript/availability/index.js:409]: ohhh this was just me trying to attempt for consistency but then I see that there are single and double in multiple places 😬

**Review by cdmo — APPROVED** [review]

**Pull request merged**